src/Avalonia.Visuals/Point.cs

```diff
@@ -267,5 +355,32 @@ public Point WithY(double y)
         {
             return new Point(_x, y);
         }
+
+        /// <summary>
+        /// Returns a new point with the opposite coordinates.
+        /// </summary>
+        /// <returns>The new point.</returns>
+        public Point Miror()
```

Comment:
Hi, is there any common practice for this? Because this is now a mi**RR**or of a relatively line y = -x

Reply:
I guess normally you would just define an unary operator `-` and call it a day.

Reply:
Nice idea, going to try it!

Reply:
But WithMirorX and WithMirorY methods still needed

Reply:
But then we can also use WithX, WithY ;) I think, that the effect will be practically same

Reply:
If use WithX, WithY, we will have this code:

```C#
NewPoint = MyPoint.WithX(-MyPoint.X)
NewPoint = MyPoint.WithY(-MyPoint.Y)
```

I think we need thouse methods

Reply:
But then a new API will appear, which is similar and very few people need (sorry, I still don't know the generally accepted examples). You can declare an extension for this

Reply:
hmm. Yes, i agree with you miror methods is unnecessary. It's can be realize to extensions

Reply:
`NewPoint = MyPoint * new Vector(-1, -1)`